client/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://json.schemastore.org/tsconfig",
-  "include": ["src/**/*", "config/**/*", "types/**/*"],
+  "include": ["src/**/*", "config/**/*", "types/**/*", "setupTests.ts"],
```

Comment:
I tested your PR removing this change and everything worked. Is there a reason to add this change? I mean, all tests seem to be executed currently without this change

Reply:
This was a mistake, sorry I didn't catch it in my self review, going to remove it now. Thanks!